components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/utils/Constants.java

```diff
@@ -88,6 +88,17 @@
     public static final String NEVER_REACHED = "Never Reached";
     public static final int DEFAULT_TIME_INTERVAL_MILLIS = 300000;
 
+    public static final String MANAGERID = "MANAGERID";
```

Comment:
What is `MANAGERID`, `WORKERID`, `HOST`, `PORT`??
use meaningful names for constants e.g. `WORKER_JVM_BLOCKED_THREADS_COUNT`.